WordPress/WordPressShareExtension/AppExtensionsService.swift

```diff
@@ -96,10 +96,7 @@ extension AppExtensionsService {
     func fetchSites(onSuccess: @escaping ([RemoteBlog]?) -> (), onFailure: @escaping FailureBlock) {
         let remote = AccountServiceRemoteREST(wordPressComRestApi: simpleRestAPI)
 
-        var filterJetpackSites = false
-        #if IS_JETPACK
-        filterJetpackSites = true
-        #endif
+        let filterJetpackSites = AppConfiguration.showJetpackSitesOnly
```

Comment:
(nit) `showJetpackSiteOnly` is not longer needed.

Reply:
I used `showJetpackSitesOnly` solely to mirror the existing code from https://github.com/wordpress-mobile/WordPress-iOS/pull/17914. Are you suggesting we remove `showJetpackSiteOnly` here, everywhere, or something else?

Reply:
I would suggest removing `AppConfiguration.showJetpackSitesOnly` since it has the same value in both wpios and jpios.

Reply:
Removed in e1a398031b7cfceb56c4705f00eeef7c0c2f4487.